src/enums.ts

```diff
@@ -95,6 +95,7 @@ export enum ScanType {
 
 export enum ScmType {
   GitHub = "GitHub",
+  Bitbucket = "Bitbucket",
```

Comment:
Wonder if we want to call this bitbucket cloud and then have one for server or we could on the provider entry point decide to do the logic of one or the other based if they user have provided another parameter like `--bitbucketServerUrl` or something like that

Reply:
let's do 'BitBucketCloud' since they are pretty different